libs/components/ag-grid/src/lib/modules/ag-grid/types/lookup-properties.ts

```diff
@@ -10,7 +10,10 @@ import {
 
 export interface SkyAgGridLookupProperties {
   /**
-   * Fires when users select the button to add options to the list.
+   * Fires when users select the button to add options to the list. If opening a modal, include
```

Comment:
```suggestion
   * Fires when users select the button to add options to the list. If the button opens a modal, include
```
Minor edit, but I think this wording is clearer. ... Also, just to make sure, is the "if" correct here? I mean, I would have assumed the button always opens a modal. Is there a second behavior where clicking the button enables the ability to add options to the list in a different way?

Reply:
Good call. Switched to `When`. The event handler is wide open, it's up to the consumer to open the modal, but that's the pattern.